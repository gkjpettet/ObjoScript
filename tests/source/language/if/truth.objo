#  False and null are false.
if false {
 System.print("bad") 
} else {
System.print("false") #  expect: false
}

if nothing { 
System.print("bad") 
} else{
 System.print("nothing") #  expect: nothing
}

#  Everything else is true.
if true then System.print(true) #  expect: true
if 0 then System.print(0) #  expect: 0
if "" then System.print("empty") #  expect: empty