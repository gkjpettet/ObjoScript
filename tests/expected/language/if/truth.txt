false
nothing
true
0
empty